examples/Connect firefly_client.ipynb

```diff
@@ -0,0 +1,269 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fc = firefly_client.FireflyClient('localhost:8080', basedir='firefly',\n",
+    "                                 channel=iv.channel)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The next two cells complete the connection of the FireflyClient."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fc.conn_id = iv.conn_id"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fc.headers = {'FF_FF-channel': fc.channel,\n",
+    "              'FF-connId': fc.conn_id}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Astropy is used here only for downloading files."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import astropy\n",
+    "img_name = astropy.utils.data.download_file('http://web.ipac.caltech.edu/staff/roby/demo/wise-m51-band2.fits')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fc.show_fits(fc.upload_file(img_name), plot_id=iv.plot_id)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Table widget example"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from firefly_widgets import TableViewer"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tv = TableViewer(position='285.396163;+33.029175;EQ_J2000',\n",
+    "                 layout=Layout(width='600px', height='400px'),\n",
+    "                tbl_id='tblwidget-1') "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tv"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Wait for the table widget to display, before executing the next cells.\n",
+    "The channel and conn_id should be the same as for the image widget"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(tv.channel, tv.conn_id, tv.tbl_id)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tbl_name = astropy.utils.data.download_file('http://web.ipac.caltech.edu/staff/roby/demo/WiseDemoTable.tbl',\n",
+    "                                            timeout=120, cache=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fc.show_table(fc.upload_file(tbl_name), tbl_id=tv.tbl_id)"
```

Comment:
I noticed that this call often results in an empty table for me. Putting fc.upload_file(tbl_name) into a separate cell helped me with this issue.